src/erlcloud_route53.erl

```diff
@@ -0,0 +1,269 @@
+%% Amazon Route 53
+
+-module(erlcloud_route53).
+
+-export([describe_zones/0, describe_zones/3]).
```

Comment:
alike other modules this need new/2/3 and config/2/3 to actually set route53 part of aws_config.